app/src/main/java/org/p2p/wallet/jupiter/statemanager/SwapStateManager.kt

```diff
@@ -133,10 +133,10 @@ class SwapStateManager(
             try {
                 while (refreshJob?.isActive == true) {
                     delay(DELAY_IN_MILLIS)
-                    handleNewAction(SwapStateAction.RefreshRoutes)
+                    onNewAction(SwapStateAction.RefreshRoutes)
```

Comment:
read what onNewAction method do, it start startRefreshJob. Why you change it?

Reply:
handleNewAction doesn't catch any throwables from Handlers